content/en/docs/concepts/architecture/controller.md

```diff
@@ -154,8 +154,8 @@ controller does.
 
 ## {{% heading "whatsnext" %}}
 
-* Read about the [Kubernetes control plane](/docs/concepts/#kubernetes-control-plane)
-* Discover some of the basic [Kubernetes objects](/docs/concepts/#kubernetes-objects)
+* Read about the [Kubernetes control plane](/docs/reference/glossary/?all=true#term-control-plane)
```

Comment:
Hi @blueabysm .
What do you think about replacing the control plane glossary term for a link to:
/docs/concepts/overview/components/
This page provides more information about the components of the control plane.

Reply:
Thank you @kbhawkey. Agree with you, Let me change it. Would it be better if the phrase "control plane" in the top quote of component page were a glossary tooltip? Perhaps talking about what the phrase "control plane" stands for at the very beginning is a little bit more helpful.

Reply:
Are you referring to the first sentence on the page, https://kubernetes.io/docs/concepts/overview/components/?
I would leave this text as is. It is created from the page `description` field.
There is a glossary tooltip for the control plane text in the third paragraph. OK?

Reply:
Yes, I saw it in the third paragraph. I mean it might be better if it could be in the first paragraph.😃
Not a big issue. BTW, I've done the modification we mentioned before.🙂